book.rb

```diff
@@ -0,0 +1,16 @@
+require './rentals'
+
+class Book
+  def initialize(title, author)
+    @title = title
+    @author = author
+    @rentals = []
+  end
+
+  attr_reader :rentals
```

Comment:
- Please ensure that the rentals attribute has both `getter and setter` methods set. Please fix this 👍 👍